packages/communication-react/review/communication-react.api.md

```diff
@@ -1291,6 +1291,8 @@ export interface _IdentifierProviderProps {
 
 // @internal
 export interface _Identifiers {
+    horizontalGalleryLeftNavButton: string;
```

Comment:
It might have been a mistake exposing `_Identifiers` through the public API (we thought @internal was OK for this, but unsure).

Let's not add to this anymore. Just directly use `data-ui-id` in the component and directly use the `data-ui-id` from e2e tests, like we already do for composites.

Thanks for trying to stick to this pattern...